Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -60,82 +61,117 @@ public class NavigationSettings {
         }
     }
 
-    private struct State {
-        static var `default`: State {
-            .init(directions: .shared,
-                  tileStoreConfiguration: .default,
-                  routingProviderSource: .hybrid,
-                  alternativeRouteDetectionStrategy: .init(),
-                  utilizeSensorData: false,
-                  navigatorPredictionInterval: nil,
-                  liveIncidentsOptions: nil,
-                  statusUpdatingSettings: nil)
-        }
+    /// All the values that you can setup NavigationSettings with.
+    public struct Values {
+        let directions: Directions
```

Comment:
Would it be more convenience for the users to have editable properties in `Value`?

From the perspective of the SDK's user initialisation still requires the call of a method with a large list of parameters